MiniProject 1/Base.lproj/Main.storyboard

```diff
@@ -1,25 +1,76 @@
+                                        <state key="normal" title="PREV">
+                                            <color key="titleColor" white="1" alpha="1" colorSpace="calibratedWhite"/>
+                                            <color key="titleShadowColor" white="1" alpha="1" colorSpace="calibratedWhite"/>
+                                        </state>
+                                        <connections>
+                                            <action selector="mainButtons:" destination="EBC-UM-TCJ" eventType="touchUpInside" id="qzI-xX-DTa"/>
+                                        </connections>
+                                    </button>
+                                </subviews>
+                                <color key="backgroundColor" white="0.0" alpha="1" colorSpace="calibratedWhite"/>
+                                <constraints>
+                                    <constraint firstAttribute="centerY" secondItem="gkf-Yf-Lf5" secondAttribute="centerY" id="8Fb-xc-9Zh"/>
+                                    <constraint firstItem="gkf-Yf-Lf5" firstAttribute="leading" secondItem="gFc-Vq-SNz" secondAttribute="leading" id="eJt-A2-A5c"/>
+                                    <constraint firstAttribute="trailing" secondItem="Agg-mz-Lf2" secondAttribute="trailing" id="otm-vv-FzV"/>
+                                    <constraint firstAttribute="centerY" secondItem="Agg-mz-Lf2" secondAttribute="centerY" id="qne-gK-4QB"/>
+                                </constraints>
+                            </view>
+                        </subviews>
+                        <color key="backgroundColor" white="1" alpha="1" colorSpace="calibratedWhite"/>
+                        <constraints>
+                            <constraint firstAttribute="trailingMargin" secondItem="gFc-Vq-SNz" secondAttribute="trailing" constant="-8" id="So8-N3-wrW"/>
+                            <constraint firstItem="1m6-Eg-WQR" firstAttribute="top" secondItem="gFc-Vq-SNz" secondAttribute="bottom" id="XTO-JU-uee"/>
+                            <constraint firstItem="gFc-Vq-SNz" firstAttribute="top" secondItem="o7J-vi-1P3" secondAttribute="bottom" constant="-20" id="ajy-7z-YVO"/>
+                            <constraint firstItem="gFc-Vq-SNz" firstAttribute="leading" secondItem="WUJ-1M-Fup" secondAttribute="leadingMargin" constant="-8" id="zLT-sM-Hw7"/>
+                        </constraints>
                     </view>
+                    <extendedEdge key="edgesForExtendedLayout"/>
+                    <connections>
+                        <outlet property="playerView" destination="gFc-Vq-SNz" id="wWl-GC-8iF"/>
+                    </connections>
                 </viewController>
-                <placeholder placeholderIdentifier="IBFirstResponder" id="dkx-z0-nzr" sceneMemberID="firstResponder"/>
             </objects>
+            <point key="canvasLocation" x="408" y="382"/>
         </scene>
     </scenes>
 </document>
```

Comment:
PREV and NEXT button should be placed as child of the "View" and not as child of the "Player View"
Constraint of the Player View are not clean (remove magic number) and introduce black border.